CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### New features
 
+* [#123](https://github.com/rubocop-hq/rubocop-performance/pull/123): Add new `Performance/AncestorsInclude` cop. ([@fatkodima][])
```

Comment:
Can you rebase with the latest master branch and move to under the `## master (unreleased)`?